notebooks/data-sources/TestGrid/metrics/get_raw_data.ipynb

```diff
@@ -0,0 +1,347 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false"># Specify the path for output metric data
# OUTPUT_DATA_PATH = "../../../../data/processed/metrics/"
</pre><p> </p><p>If this notebook is only ever to be used with Ceph, then you should remove this instead of commenting it out. Otherwise, you should add the save-to-disk and in-automation behavior to this notebook as well.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>